### PR TITLE
uvesh

### PR DESCRIPTION
**Pull request by UveshRajwani**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/uvesh.is-a.dev.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "uvesh's .is-a.dev domain",
+    "owner": {
+      "username": "UveshRajwani",
+      "email": "uveshrajwani786@gmail.com"
+    },
+    "record": {
+       "A": [
+      "167.86.86.55"
```
Comment (WilliamDavidHarrison) [inline domains/uvesh.is-a.dev.json:9]: I just get the default Nginx landing page on port 80. What port is your website on?

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please remove the `.is-a.dev` from the file name and see the review comment below.

File: domains/uvesh.is-a.dev.json (hunk 1 above)
Comment (UveshRajwani) [reply domains/uvesh.is-a.dev.json:9]: yes i am making the website for now i have the default page

Comment (WilliamDavidHarrison) [reply domains/uvesh.is-a.dev.json:9]: @UveshRajwani You must have a website to get a domain.

**Comment by TheBotlyNoob** [comment]
I'm closing this because of the large amounts of pull requests. You can make another PR later with fixes if you want.

**Pull request closed without merging**